src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/[[...slug]]/page.tsx

```diff
@@ -106,11 +107,21 @@ export default async function DashboardPage(props: Props) {
     return redirect("/user/welcome");
   }
 
-  const latestScan = await getScanResultsWithBroker(
+  const enabledFeatureFlags = await getEnabledFeatureFlags({
```

Comment:
we should wrap the logic here with an `if` check (safer):
```
if (not prod && feature flag enabled for this email) {
  scanResults = getMock()
} else {
  scanResults = getReal()
}
```

This way we 1) only go to the db once 2) logic seems clearer to read